rental_pricelist/views/product_view.xml

```diff
@@ -56,7 +56,6 @@
                                         />
                                         <field name="compute_price" invisible="1" />
                                         <field name="applied_on" invisible="1" />
-                                        <field name="product_id" invisible="1" />
```

Comment:
Why do you delete this line?

Reply:
in v14 due to view changes, either we need to remove it or add more fields to adjust our field `product_id`